lua/telescope/_extensions/git_diffs.lua

```diff
@@ -20,17 +20,15 @@ local function diffview(prompt_bufnr)
     return
   end
 
-  local new = string.sub(selections[1].value, 1, 8)
```

Comment:
You could do this instead if you want keep the `--graph` flag
`local new = string.sub(selections[1].ordinal, 2, 9)`